tools/test_rouge.py

```diff
@@ -0,0 +1,61 @@
+# -*- encoding: utf-8 -*-
+import argparse
+import os
+import time
+import pyrouge
+import shutil
+
+
+def test_rouge(cand_file, ref_file):
```

Comment:
Just wanted to check this gets the same score as with other methods? It calls the rouge perl behind the scenes?

Reply:
Not exactly. I use pyrouge instead of pythonrouge as my backend. The scores will be a little different but in same confidence interval, since they use different default options to call ROUGE-1.5.5.pl. For example, files2rouget use  ["-a", "-m", "-2", "4","-n", "3"], but pyrouge use ['-a', '-n', 4, '-x', '-2', 4, '-u', '-c', 95,  '-r', 1000, '-f', 'A', '-p', 0.5, '-t', 0, '-d']. I just follow See et al. 2017 to use this package. Should I change the backend as files2rouge?

Reply:
No, it's fine, but document this exactly.